slack_sdk/web/async_client.py

```diff
@@ -146,6 +146,97 @@ async def admin_apps_restricted_list(self, **kwargs) -> AsyncSlackResponse:
             "admin.apps.restricted.list", http_verb="GET", params=kwargs
         )
 
+    async def admin_barriers_create(
+        self,
+        *,
+        barriered_from_usergroup_ids: Union[str, Sequence[str]],
+        primary_usergroup_id: str,
+        restricted_subjects: Union[str, Sequence[str]],
+        **kwargs
+    ) -> AsyncSlackResponse:
+        """Create an Information Barrier.
+
+        Args:
+            barriered_from_usergroup_ids (str or list): A list of IDP Groups ids that the
+                primary usergroup is to be barriered from.
+            primary_usergroup_id (str): The id of the primary IDP Group.
+            restricted_subjects (str or list): What kind of interactions are blocked by this
+                barrier? For v1, we only support a list of all 3, eg im, mpim, call
+
+        """
+        kwargs.update({"primary_usergroup_id": primary_usergroup_id})
+
+        if isinstance(barriered_from_usergroup_ids, list):
```

Comment:
This is an existing issue but I found that I should have changed this when I changed the type hints in method signature. I will come up with a PR changing others. Can you do the same in this PR?
```suggestion
        if isinstance(barriered_from_usergroup_ids, Sequence):
```

```python
>>> from typing import Sequence, Tuple
>>> a, b = ["a", "b"], ("a", "b")
>>> ",".join(a)
'a,b'
>>> ",".join(b)
'a,b'
>>> isinstance(a, list)
True
>>> isinstance(a, Sequence)
True
>>> isinstance(a, Tuple)
False
>>> isinstance(b, list)
False
>>> isinstance(b, Sequence)
True
>>> isinstance(b, Tuple)
True
```

Reply:
I found that type checking needs to be more specific:

```python
>>> s = "C123,C234"
>>> from typing import Sequence, Tuple
>>> isinstance(s, Sequence)
True
>>> isinstance(s, (list, Tuple, set))
False
```

Reply:
I've updated other methods to check both list and tuple. this method should be updated accordingly.

https://github.com/slackapi/python-slack-sdk/pull/896